openfisca_uk/variables/benunit/benefits/computed.py

```diff
@@ -0,0 +1,348 @@
+from openfisca_core.model_api import *
+from openfisca_uk.entities import *
+import numpy as np
+
+
+class income_support(Variable):
+    value_type = float
+    entity = BenUnit
+    label = u"Income Support amount received per week"
+    definition_period = ETERNITY
+
+    def formula(benunit, period, parameters):
+        younger_age = benunit("younger_adult_age", period)
+        older_age = benunit("older_adult_age", period)
+        personal_allowance = (
```

Comment:
should this be its own concept? and/or could you describe the logic in a comment, given it's complexity?